server/migrations/versions/c012fa6b13a9_sampledbatchdraw_contest_id.py

```diff
@@ -0,0 +1,57 @@
+# pylint: disable=invalid-name
+"""SampledBatchDraw.contest_id
+
+Revision ID: c012fa6b13a9
+Revises: 83bc53b14021
+Create Date: 2024-02-09 15:38:22.522331+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "c012fa6b13a9"
+down_revision = "83bc53b14021"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Modeled this migration after the [very similar migration for adding a contest ID to the `SampledBallotDraw` model](https://github.com/votingworks/arlo/blob/73d5140148604687d2de4de40d5056d41f99a178/server/migrations/versions/5b7164faeda5_sampledballotdraw_contest_id.py)